www/creating-zooming-images.html

```diff
@@ -182,7 +182,14 @@ <h2>Creating Zooming Images</h2>
         <td>Free Google Cloud Dataflow template allowing large amounts of images to be processed in parallel</td>
         <td><a href="/examples/tilesource-zoomify/">Zoomify</a></td>
     </tr>
-    
+    <tr>
+        <td colspan="3"><h4>Iris</h4></td>
```

Comment:
Thank you for adding this here! Please move it so it's alphabetically located below IIIF.